lib/features/docker_save.js

```diff
@@ -65,10 +67,11 @@ export default class DockerSave {
       zipStream.on('error', reject);
     });
     let expiration = taskcluster.fromNow(task.runtime.dockerSave.expiration);
+    expiration = new Date(Math.min(expiration, new Date(task.task.expires)));
     let stat = await fs.stat(pathname);
 
-    await uploadToS3(task,
-      fs.createReadStream(pathname),
+    await uploadToS3(task.runtime.queue, task.status.taskId, task.runId,
+                     fs.createReadStream(pathname),
```

Comment:
any reason for indenting this, but not the other arguments passed in below this line?